src/plugins/intel_cpu/src/nodes/gather.cpp

```diff
@@ -972,7 +972,7 @@ void Gather::resolveInPlaceEdges(Edge::LOOK look) {
     auto& config = selected_pd->getConfig();
     size_t inplaceInpIndx = selected_pd->getConfig().outConfs[outputPort].inPlace();
     const auto baseDim = inputShapes.front().getDims()[axis];
-    OPENVINO_ASSERT(baseDim != Shape::UNDEFINED_DIM,
+    CPU_NODE_ASSERT(baseDim != Shape::UNDEFINED_DIM,
```

Comment:
Please remove the node type and name from the message, as the macro add this info as a prefix automatically. Here and below.

Reply:
Done